### ISSUE TITLE
inputPlaceholderText fails to meet accessibility rules as the contrast ratio to background color is less then 4.5:1.

### ISSUE THREAD

**Issue by eloisepeng** [microsoft/fluentui#17661]
UI Fabric Theme Designer's accessibility checker fails to report color contrast issues. inputPlaceholderText to background contrast ratio is less than 4.5:1 as required for any colors other than text color in black. Also, for text color set to black, we observed a contrast color for inputPlaceholderText is 11.725:1, which is too high considering it's a placeholder color. 

Repro steps:

1. open [UI Fabric Theme Designer](https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/9318/merge/theming-designer/index.html)
2. extend Inputs (14)
3. open Accessibility Insights and inspect inputPlaceholderText color
4. inputPlaceholderText color fails the contrast ratio requirement (<4.5:1) 
5. if we change the Background color to black, we observed inputPlaceholderText ratio is 11.725:1, which seems too high

#### Expected behavior:

inputPlaceholderText : Background color >= 4.5:1, we expected this color to be lowest contrast ratio we can get as it's a placeholder color

![image](https://user-images.githubusercontent.com/32313451/113249309-559a4500-9273-11eb-9b9c-1f392622a148.png)

**Comment by eloisepeng** [comment]
MS Office is using this theme color for our search input placeholder. we have a bug report that depends on this fix, please help me contact the right people and attach the correct labels to help.

**Comment by gouttierre** [comment]
@smhigley  - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @miroslavstastny

**Comment by smhigley** [comment]
The link and screenshot above look like they're from an outdated Theme Designer PR preview link from 2019. The current [Theme Designer](https://fluentuipr.z22.web.core.windows.net/heads/master/theming-designer/index.html) is using a darker color for placeholder that meets the minimum contrast:

![screenshot of the theme designer placeholder color and a devtools eyedropper showing a 6.46:1 contrast ratio](https://user-images.githubusercontent.com/3819570/130117359-8bd4421e-cac3-4a03-99ea-708c4bb083ad.png)

The actual placeholder text on inputs within the current Theme Designer also meets contrast correctly. It looks to me like this is an outdated issue, but please let me know if I'm missing anything :)
